lab-kyle/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+exports.greet = function(name) {
+  const input = name || 'world';
```

Comment:
Since you're implementation only executes this module's function when its parameter name is truthy, I suggest you remove line 4 and replace "input" on line 5 with "name".